CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# [VuFind Developer Manual](https://vufind.org/wiki/development)
+
+## Prerequisites
+
+ - [Community Code of Conduct](https://github.com/vufind-org/vufind/CODE_OF_CONDUCT.md)
+ - [Coding Standards](https://vufind.org/wiki/development:coding_standards) - How to conform to VuFind's style.
+ - [Recommended Tools](https://vufind.org/wiki/development:recommended_tools) - Suggestions for successful VuFind development.
+ - [Making Pull Requests](https://vufind.org/wiki/development:making_pull_requests)
+ 
+## About VuFind
+
+ - [Architecture Notes](https://vufind.org/wiki/development:architecture) - Information on VuFind's design and dependencies.
+ - [How-Tos](https://vufind.org/wiki/development:howtos) - Step-by-step instructions on various VuFind development tasks.
+ - [Plugins](https://vufind.org/wiki/development:plugins) - Information on the types of plug-ins used to extend VuFind functionality.
+ - [Profiling](https://vufind.org/wiki/development:profiling) - Information on profiling VuFind's code to improve performance.
+ - [Testing](https://vufind.org/wiki/development:testing) - Notes on testing VuFind.
+ - [Troubleshooting](https://vufind.org/wiki/development:troubleshooting) - Notes on troubleshooting and debugging VuFind.
+ - [APIs](https://vufind.org/wiki/development:apis) - APIs that VuFind provides for searching and other functionality.
+ 
+## VuFind Planning
+ - [Bi-weekly developers call](https://vufind.org/wiki/developers_call)
```

Comment:
This is now a monthly community call, so we should be careful about the language here. :-) Where did this link come from? Was this copied and pasted from somewhere else? If so, we need to fix that too!

Reply:
This link is [right from the wiki](https://vufind.org/wiki/development).

Reply:
I've fixed it in the wiki -- thanks!